packages/algo-ts/src/arc4/encoded-types.ts

```diff
@@ -311,6 +334,10 @@ export class StaticBytes<TLength extends number = 0> extends Arc4ReadonlyArray<B
   get native(): bytes {
     throw new NoImplementation()
   }
+
+  copy(): StaticBytes<TLength> {
```

Comment:
Static bytes is immutable so it doesn't require a .copy

Reply:
It has been reverted.